style.css

```diff
@@ -415,18 +416,19 @@ textarea {
 
 .button {
   width: 160px;
-  border: 2px solid #7f8cff;
+  border: 2px solid #2838cb;
```

Comment:
- [ ] Dear @team, You did great implementing the most of required changes. one thing is left regarding the contrast ratio as the previous code reviewer mentioned. Please address this to help your site be more accessible. 

### Please [use this link](https://www.aditus.io/button-contrast-checker/htmlpreview-github-io-2022-11-08-at-15-24-32-909?failing) to check for the contrast ratio.

![Capture](https://user-images.githubusercontent.com/69077061/200607194-fb4075e3-ea23-48e9-a282-7d574d691825.PNG)

Reply:
- dear @quavo19, You are almost done. For the pull request description, I suggest you refer to [this pull request](https://github.com/Vgonma/PersonalPortfolio/pull/8). There are some details missing there also but this should be good for you to refer to. 

- And for the contrast, you just have to fix one thing. the `Get in touch` button. please [use this link](https://www.aditus.io/button-contrast-checker/htmlpreview-github-io-2022-11-08-at-15-24-32-909?failing)to review your result before submitting it for a review.